Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/EmojiContainerViewController.swift

```diff
@@ -154,30 +155,9 @@ class EmojiContainerViewController: UIViewController {
     }
 
     private func setupGifLayout() {
-        view.insertSubview(gifView, at: 0)
-        gifView.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
-        gifView.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
-        gifView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
-        gifView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
-
-        giphy.clipsPreviewRenditionType = .preview
-        addChild(giphy)
-        gifView.addSubview(giphy.view)
-        searchView.placeholder = "Search Gif"
-        gifView.addSubview(searchView)
-        searchView.pin(anchors: [.top, .leading, .trailing], to: gifView)
-        searchView.searchBarStyle = .prominent
-        searchView.delegate = self
-        giphy.view.translatesAutoresizingMaskIntoConstraints = false
-        giphy.view.leftAnchor.constraint(equalTo: gifView.safeLeftAnchor).isActive = true
-        giphy.view.rightAnchor.constraint(equalTo: gifView.safeRightAnchor).isActive = true
-        giphy.view.topAnchor.constraint(equalTo: searchView.bottomAnchor).isActive = true
-        giphy.view.bottomAnchor.constraint(equalTo: gifView.safeBottomAnchor).isActive = true
-        giphy.didMove(toParent: self)
-        let trendingGIFs = GPHContent.trending(mediaType: .gif)
-        giphy.content = trendingGIFs
-        giphy.delegate = self
-        giphy.update()
+        let gifVc = GifViewController(with: false)
+        addChildViewController(gifVc, embedIn: view)
+        return
```

Comment:
@mdhanif-simformsolutions  Please remove `return` it is not required here.

Reply:
Updated this. thank you.